src/firefly/js/visualize/task/ImageOverlayTask.js

```diff
@@ -2,22 +2,33 @@
  * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
  */
 
-import {logError} from '../util/WebUtil.js';
-import ImagePlotCntlr, {makeUniqueRequestKey, IMAGE_PLOT_KEY, dispatchPlotMask} from './ImagePlotCntlr.js';
-import {primePlot, getOverlayByPvAndId, getPlotViewById} from './PlotViewUtil.js';
-import {PlotState, RotateType} from './PlotState.js';
-import {RequestType} from './RequestType.js';
-import {ZoomType} from './ZoomType.js';
-import {clone} from '../util/WebUtil.js';
-import {WebPlot} from './WebPlot.js';
-import {callGetWebPlot} from '../rpc/PlotServicesJson.js';
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
 
+import {get} from 'lodash';
+import {logError} from '../../util/WebUtil.js';
+import ImagePlotCntlr, {makeUniqueRequestKey, IMAGE_PLOT_KEY, dispatchPlotMask, dispatchZoom} from '../ImagePlotCntlr.js';
+import {UserZoomTypes} from '../ZoomUtil.js';
+import {primePlot, getOverlayByPvAndId, getPlotViewById, getOverlayById} from '../PlotViewUtil.js';
+import {PlotState, RotateType} from '../PlotState.js';
+import {RequestType} from '../RequestType.js';
+import {ZoomType} from '../ZoomType.js';
+import {clone} from '../../util/WebUtil.js';
+import {WebPlot} from '../WebPlot.js';
+import {callGetWebPlot} from '../../rpc/PlotServicesJson.js';
+
+const colorList= [
+    '#FF0000','#00FF00', '#0000FF', '#91D33D',
+    '#AE14E0','#FFC0CB', '#EBAA38', '#F6E942',
+    '#00E8FF','#8B572A', '#B8E986', '#4A90E2',
+    '#BD10E0','#E0107F', '#B9F81C', '#F19301',
+];
 
 function *getColor()  {
-    const autoColor= ['#FF0000','#00FF00','#0000FF','#91D33D', '#AE14DB','#FF0000', '#EBAA38', '#F6E942'];
     var nextColor=0;
     while (true) {
-        yield autoColor[nextColor % autoColor.length];
+        yield colorList[nextColor % colorList.length];
         nextColor++;
     }
 }
```

Comment:
Not using it any more but I don't want to delete since I can see using it in the future